gear/user_management/src/python/user_app/run.py

```diff
@@ -125,7 +134,9 @@ def run(self, context: GearToolkitContext) -> None:
                                              source=self.__email_source)),
                 registry=UserRegistry(api_instance=DefaultApi(comanage_client),
                                       coid=self.__comanage_coid),
-                force_notifications=self.__force_notifications)
+                force_notifications=self.__force_notifications,
+                parameter_store=self.__parameter_store,
```

Comment:
see note about changing admin group in review. these parameters shouldn't be needed here